classes/class-p4-search.php

```diff
@@ -366,12 +366,18 @@ protected function get_posts( $paged = 1 ) : array {
 				foreach ( $posts as $post ) {
 					if ( P4_Post_Archive::POST_TYPE === $post->post_type ) {
 						$archive_post               = new stdClass();
+						$archive_post->id           = $post->ID;
```

Comment:
Do we need to keep post id in archive data?

Reply:
Not necessarily but might make sense to keep it as the other post types also have this set. During development I at first forgot to add this and accessed the ID in the template and it actually didn't fail but produced a wrong link, which is easy to overlook. So I guess that would be prevented from happening again by keeping it.